### PR TITLE
Track ip

### PR DESCRIPTION
**Pull request by Anna-Livia**
Connected to openfisca/openfisca-france#821

This PR adds the possibility to track the API visitor's IP address.

### CODE REVIEW

File: tests/test_piwik.py [hunk 1]
```diff
@@ -1,21 +1,19 @@
 # -*- coding: utf-8 -*-
-
-import time
-
-from nose.tools import assert_equal
-
 from openfisca_tracker.piwik import PiwikTracker, BUFFER_SIZE
 
-TRACKER_URL = 'https://stats.data.gouv.fr/piwik.php'
+TRACKER_URL = 'http://stats.data.gouv.fr/piwik.php'
```
Comment (fpagnoux) [inline tests/test_piwik.py]: Why switching to HTTP ? Sending a secret token through HTTP seems risky...

File: tests/test_piwik.py [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
 TRACKER_IDSITE = 4
+TRACKER_AUTH = ""
```
Comment (fpagnoux) [inline tests/test_piwik.py]: Does the test pass with this value ?

**Review by fpagnoux — CHANGES_REQUESTED** [review]
Not tested, but here are comments on the code

**Review by fpagnoux — CHANGES_REQUESTED** [review]
Not tested, but here are comments on the code

File: tests/test_piwik.py (hunk 2 above)
Comment (Anna-Livia) [reply tests/test_piwik.py]: Yep :)

**Review by fpagnoux — APPROVED** [review]

File: openfisca_tracker/piwik.py [hunk 3]
```diff
@@ -33,14 +36,15 @@ def send(self):
...
+                headers = {"Accept": "application/json"},
+                params = json.dumps({"requests": self.requests, "token_auth": self.token_auth}),
                 callback = default_callback
                 )
             self.requests = []
 
-    def track(self, action_url):
-        tracked_request = "?idsite={}&url={}&rec=1".format(self.idsite, action_url)
+    def track(self, action_url, action_ip=""):
+        log.debug(action_url)
```
Comment (fpagnoux) [inline openfisca_tracker/piwik.py]: Are we sure we want to log this, especially without any additional context ?

**Pull request merged**